frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -0,0 +1,294 @@
+                            <Button type="button" variant="outline" onClick={() => setAmount("10000")}>✨ 10000</Button>
+                          </div>
+                        </div>
+
+                      </div>
+                      <div className="grid grid-cols-4 items-center gap-4">
+                        <Label htmlFor="comment" className="text-right">
+                          Name <span className="font-normal text-muted-foreground">(optional)</span>
+                        </Label>
+                        <Input
+                          id="sender-name"
+                          value={senderName}
+                          onChange={(e) => setSenderName(e.target.value)}
+                          placeholder={`Nickname, npub, @twitter, etc.`}
+                          className="col-span-3" />
+                      </div>
+                    </div>
+
+                    <AlertDialogFooter>
+                      <AlertDialogCancel>Cancel</AlertDialogCancel>
+                      <LoadingButton
+                        type="submit"
+                        disabled={!!isSubmitting}
+                        loading={isSubmitting}
+                      >
+                        Complete Setup
+                      </LoadingButton>
+                    </AlertDialogFooter>
+                  </form>
+                </AlertDialogContent>
+              </AlertDialog></>
+          }
+          {subscription && <>
+            <Card className="w-full">
+              <CardHeader>
+                <CardTitle className="flex flex-row justify-between">
+                  <div className="flex">
+                    <CheckCircle2 className="w-4 h-4 mr-2" />
+                    You are an Alby Supporter!
+                  </div>
+                </CardTitle>
+              </CardHeader>
+              <CardContent>
+                <div className="flex flex-row justify-between">
+                  <div>
+                    <div>
+                      <p className="text-muted-foreground text-sm">Total contributions</p>
+                      <p className="text-xl font-medium">
+                        {new Intl.NumberFormat().format(
+                          Math.floor(1000000 / 1000)
+                        )}{" "}
+                        sats
+                      </p>
+                    </div>
+                  </div>
+                  <div>
+                    <p className="text-muted-foreground text-sm">Amount per month</p>
+                    <p className="text-xl font-medium">
+                      {new Intl.NumberFormat().format(
+                        Math.floor(1000000 / 1000)
+                      )}{" "}
+                      sats
+                    </p>
+                  </div>
+                </div>
+              </CardContent>
+            </Card>
+          </>}
+        </div>
+      </div >
```

Comment:
weird formatting here

Also we need a button for the user to go back home.

Maybe also write "You can view your support payments or cancel at any time by going to the connections page and opening ZapPlanner - Alby Hub."